contracts/coordinator/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "0.1.0",
+        "changes": [
+            {
+                "note": "Added Coordinator Registry",
+                "pr": 1675
+            }
+        ]
+    },
```

Comment:
It looks like we never published `0.0.1`, let's roll these into one CHANGELOG entry.